test/integration/nss-tests.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+
+set -xo pipefail
+
+# SPDX-License-Identifier: BSD-2-Clause
```

Comment:
I know this is a copy + paste issue from my mistake, can you make this line no 2.

Reply:
Fixed as well.